Formula/zsh.rb

```diff
@@ -22,6 +22,7 @@ class Zsh < Formula
 
   deprecated_option "disable-etcdir" => "without-etcdir"
 
+  depends_on "ncurses"
```

Comment:
Please add a `revision 1` line, after `sha256`.

Reply:
Added the `revision 1` after the main `sha256`. Also rebased on master fwiw.